todo_app/browser_tests/chrome_test.py

```diff
@@ -0,0 +1,53 @@
+import os, requests, json
+import pytest
+import todo_app.app
+from selenium import webdriver
+from todo_app.trello_api import create_trello_board, delete_trello_board
+from threading import Thread
+import dotenv
+
+
+@pytest.fixture(scope='module')
```

Comment:
I'm not sure how you're running these but they didn't quite work on my machine by default; your good work in getting both firefox and chrome tests up has unfortunately bitten you! For me the two test_app functions appear to interfere with each other, so that when ran together, whichever test is second fails. For a lazy way out I'm happy for you to remove one of the two browser tests, or if we share the test_app definition (by deleting one version, setting the survivor's scope to `session` and importing it in the other file) that should resolve the issue properly. You should be able to reproduce by running the tests with `poetry run pytest todo_app/browser_tests`

Reply:
For now I have cheated by adding comments to the readme that you can only run these tests separately... hopefully this is OK to move me past this module (I'd like to keep both browser tests in my code for now)?  If I get time in the future I will look into shared definition (Hugh did mention this to me when he helped me with some issues)

Reply:
Happy with that for now